src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/auth/GithubAuthenticationTests.kt

```diff
@@ -74,13 +74,13 @@ class GithubAuthenticationTests : IntegrationTest()
     }
 
     @Test
-    fun `authentication fails if token does not have org reading scope`()
+    fun `authentication fails if token does not have user reading scope`()
     {
         // this is a PAT for a test user who only has access to a test org with no repos
         // reversed so GitHub doesn't spot it and invalidate it
-        val tokenWithoutOrgReadingScope = "285d9b1b6620ab4dfd6c403b29451d52aa38a158".reversed()
+        val tokenWithoutUserReadingScope = "285d9b1b6620ab4dfd6c403b29451d52aa38a158".reversed()
```

Comment:
In the process of getting these tests to pass I found that Github auth was not in fact working when an org was specified. I first thought it was because of missing scope as indicated by this test. In fact we don't need org reading scope but I corrected this test.